openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +520,17 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
Perhaps this should live in helpers instead of api.py to make it clearer that this is not a user-facing API. It's also where the rest of the profile image methods live.